Hymns/Display/HymnLyricsViewModel.swift

```diff
@@ -0,0 +1,26 @@
+import Foundation
+import Resolver
+
+class HymnLyricsViewModel: ObservableObject {
+    
+    @Published var lyrics: [Verse]? = [Verse]()
+    
+    @Injected private var hymnsRepository: HymnsRepository
```

Comment:
Can you explain @Injected? I get that it is from Resolve?

Reply:
Admittedly this is a bit of a magical invocation. Basically what it does under the hood is that it creates getters/setters, and the DI framework will set the variable behind the scenes, so you will just have it.

I'm unsure if we want to go this route or if we want to fully go the constructor injection route. It is less code in terms of boilerplate and set-up, but we'll see if it gives us problems in the future.